README.md

```diff
@@ -6,17 +6,21 @@ two EVM-based blockchain networks by collecting bridge validators signatures to
 
 Currently the contracts supports two types of relay operations:
 * to tokenize native coins circulating in one blockchain network (Home) into an ERC20 token in another network (Foreign);
-* to swap a token presented by an ERC20 contract in Home network to an existing ERC20 token in Foreign network.
+* to swap a token presented by an existing ERC20 contract in Foreign network to an ERC20 token in Home network, one pair of bridge contracts correspond to one pair of ERC20 tokens.
 
-POA bridge consist of several main parts:
-* Home Bridge is a smart contract that should be deployed in POA.network;
+This version of contract is intended to be work with [the bridge process implemented on NodeJS](https://github.com/poanetwork/bridge-nodejs).
+Please refer to the bridge process documentation to deploy it and configure.
+
+POA bridge contracts consist of several main parts:
+* Home Bridge is a smart contract that should be deployed in POA.Network;
 * Foreign Bridge is a smart contract that should be deployed in Ethereum Mainnet;
 * Depending on type of relay operations the following components are used as well:
   * in `NATIVE-TO-ERC` mode: ERC20 token (in fact, ERC677 extension is used) should be deployed on Foreign network;
   * in `ERC-TO-ERC` mode: ERC20 token (in fact, ERC677 extension is used) should be deployed on Home network;
```

Comment:
* in `ERC-TO-ERC` mode: **The** ERC20 token (in fact, ERC677 extension is used) should be deployed on **the** Home network;